### ISSUE TITLE
Provide useful logging for developers

### ISSUE THREAD

**Issue by jim** [City-Bureau/city-scrapers#239]
The default logging we have now is very noisy and very difficult to interpret. We should replace it with something that is understandable and useful.

This might mean raising the default logger level to quiet things down and then adding our own logging statements in the right places.

**Comment by easherma** [comment]
Partially addressed by #272, which makes a JSON feed export option available.

**Comment by pjsier** [comment]
Updating the scrapers to use `Item` subclasses makes the logging a little more straightforward:

```
2019-10-03 11:22:16 [scrapy.core.scraper] DEBUG: Scraped from <200 https://www.edgewater.org/ssa-26/commissionmeetings/>
{'all_day': False,
 'classification': 'Commission',
 'description': '',
 'end': datetime.datetime(2017, 7, 14, 17, 0),
 'id': 'chi_ssa_26/201707141500/x/commission',
 'links': [{'href': 'https://www.edgewater.org/wp-content/uploads/2014/03/7-14-17-minutes.pdf',
            'title': 'Minutes'}],
 'location': {'address': '1210 W Rosedale Ave Chicago, IL 60660',
              'name': 'Edgewater Chamber of Commerce'},
 'source': 'https://www.edgewater.org/ssa-26/commissionmeetings/',
 'start': datetime.datetime(2017, 7, 14, 15, 0),
 'status': 'passed',
 'time_notes': '',
 'title': 'Commission'}
```

Scrapy also has JSON output from the command line built-in with:

```
scrapy crawl chi_ssa_26 -o output.json
```
